feeds/scheduler/feed_group.go

```diff
@@ -11,12 +12,23 @@ import (
 	"github.com/ossf/package-feeds/publisher"
 )
 
+var (
+	errPoll = errors.New("error when polling for packages, see log for details")
```

Comment:
I'd prefer we avoided mentioning to check the logs in the error itself as this could end up being processed as a go error. "see log for details" should be specific to the HTTP error messaging.

Reply:
Agreed, I've removed "see log for details" from the generic error and added it explicitly to the http error string